tests/rootfs-image-v2

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Following the other comment, this change should be removed.

Reply:
Fixed in v2